homeassistant/components/http/ban.py

```diff
@@ -34,6 +34,9 @@
 KEY_FAILED_LOGIN_ATTEMPTS: Final = "ha_failed_login_attempts"
 KEY_LOGIN_THRESHOLD: Final = "ha_login_threshold"
 
+EVENT_FAILED_LOGIN = "login_failed"
+EVENT_BAN_IP = "ban_ip"
```

Comment:
I think `ip_banned` would be a better event name, and would match better with `login_failed`

Reply:
Agreed. Made the change. Thanks!